pyvelox/test/test_signatures.py

```diff
@@ -41,9 +41,11 @@ def test_function_signature(self):
         concat_signatures = presto_signatures["concat"]
         self.assertTrue(len(concat_signatures) > 0)
         # Array functions are registered first, then string functions.
+        concat_signatures.sort(key=lambda sig: sig.__str__())
         self.assertEqual(str(concat_signatures[0].return_type()), "array(__user_T1)")
```

Comment:
> self.assertEqual(str(concat_signatures[0].return_type()), "array(__user_T1)")

Does this code assume that the first signature of "concat" function is an array(__user_T1)? Seems fragile, no?